frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -114,159 +115,226 @@ export function SurveyForm({ id }: { id: string }): JSX.Element {
-                                    question.type === SurveyQuestionType.MultipleChoice) && (
-                                    <div className="flex flex-col gap-2">
-                                        <Field name="choices" label="Choices">
-                                            {({ value, onChange }) => (
-                                                <div className="flex flex-col gap-2">
-                                                    {(value || []).map((choice: string, index: number) => (
-                                                        <div className="flex flex-row gap-2" key={index}>
+                                <LemonCollapse
+                                    defaultActiveKey="question"
+                                    panels={[
+                                        {
+                                            key: 'question',
+                                            header: (
+                                                <div className="flex flex-row w-full items-center justify-between">
+                                                    <b>{question.question}</b>
+                                                    {survey.questions.length > 1 && (
+                                                        <LemonButton
+                                                            icon={<IconDelete />}
+                                                            status="primary-alt"
+                                                            data-attr={`delete-survey-question-${index}`}
+                                                            onClick={() => {
+                                                                setSurveyValue(
+                                                                    'questions',
+                                                                    survey.questions.filter((_, i) => i !== index)
+                                                                )
+                                                            }}
+                                                            tooltipPlacement="topRight"
+                                                        />
+                                                    )}
+                                                </div>
+                                            ),
+                                            content: (
+                                                <>
+                                                    <Field name="type" label="Question type" className="max-w-60">
+                                                        <LemonSelect
+                                                            onSelect={(newType) => {
+                                                                const isEditingQuestion =
+                                                                    defaultSurveyFieldValues[question.type].questions[0]
```

Comment:
setting defaults isn't working correctly it seems, because it's still referencing the first question in here by index, rather than the current one

Reply:
Something similar going wrong with the appearance

![2023-10-02 10 40 01](https://github.com/PostHog/posthog/assets/7115141/4ad5aae7-bf0e-4555-ae53-f77f7be1cc6b)

I guess fine to fix this in a follow up, as long as multiple qs are flagged only to us